snuba/web/rpc/v1/resolvers/R_eap_spans/resolver_get_trace.py

```diff
@@ -41,54 +41,13 @@
     attribute_key_to_expression_eap_items,
 )
 
-NORMALIZED_COLUMNS_TO_INCLUDE = [
-    col.name
-    for col in get_entity(EntityKey("eap_spans")).get_data_model().columns
-    if col.name
-    not in [
-        "retention_days",
-        "sign",
-        "attr_str",
-        "attr_num",
-        "span_id",
-        "timestamp",
-        "time",
-    ]
-]
-
-
 NORMALIZED_COLUMNS_TO_INCLUDE_EAP_ITEMS = [
     "organization_id",
     "project_id",
     "trace_id",
-    "sampling_weight",
+    "sampling_factor",
 ]
 
-# Map of eap_items attributes to eap_spans attributes to preserve backwards compatibility
-EAP_ITEMS_ATTRIBUTE_MAP = {
-    "sentry.raw_description": "name",
-    "sentry.transaction": "segment_name",
-    "sentry.start_timestamp_precise": "start_timestamp",
-    "sentry.end_timestamp_precise": "end_timestamp",
-    "sentry.duration_ms": "duration_ms",
-    "sentry.event_id": "event_id",
-    "sentry.exclusive_time_ms": "exclusive_time_ms",
-    "sentry.is_segment": "is_segment",
-    "sentry.parent_span_id": "parent_span_id",
-    "sentry.profile_id": "profile_id",
-    "sentry.received": "received",
-    "sentry.segment_id": "segment_id",
-}
-
-# Attributes that we store in eap_items but not eap_spans
-ATTRIBUTES_TO_SKIP_FROM_EAP_ITEMS = [
-    "event_id",
-    "received",
-]
-
-# Attributes to hexify
-HEX_COLUMNS = ["parent_span_id", "segment_id"]
```

Comment:
if we are removing these, doesn't that assume that everything will be writing to the eap_items consumer immediately?

Reply:
This was meant to preserve some compatibility for this endpoint when we had the split `eap_spans`/`eap_items`. We definitely do not want to return aliases for any attributes, especially in a new endpoint.

What is written right now are attributes with the prefix `sentry.` and the product team will strip or do whatever they need to do to show it.

This endpoint is not used in production yet.